Makefile

```diff
@@ -16,3 +16,6 @@ benchmark:
 checks:
 	go vet ${PKGS}
 	staticcheck ${PKGS}
+
```

Comment:
How about adding the linter as the last step of this target?
```suggestion
	make lint

```

Reply:
sounds good. I'll make the change